CMakeLists.txt

```
@@ -74,9 +74,9 @@ set(flags_to_test
     -D_FORTIFY_SOURCE=2)
 if(NOT MSVC)
     list(INSERT flags_to_test 0 -Wall)
-        option(NATIVE "Build for native performance (march=native)" ON)
+        option(NATIVE "Build for native performance (march=native)")
         if(NATIVE)
-        list(APPEND flags_to_test -march=native)
+        list(APPEND flags_to_test -march=sse4.1)
```

```suggestion
        list(APPEND flags_to_test -march=native)
```
`-march=sse4.1` is still not an actual option, plus this is supposed to be reserved for the native option (I messed up on the indentation)